AK/Base64.h

```diff
@@ -13,14 +13,13 @@
 
 namespace AK {
 
-size_t calculate_base64_decoded_length(StringView);
+[[nodiscard]] size_t calculate_base64_decoded_length(StringView);
 
-size_t calculate_base64_encoded_length(ReadonlyBytes);
+[[nodiscard]] size_t calculate_base64_encoded_length(ReadonlyBytes);
 
-ErrorOr<ByteBuffer> decode_base64(StringView);
-
-String encode_base64(ReadonlyBytes);
+[[nodiscard]] ErrorOr<ByteBuffer> decode_base64(StringView);
```

Comment:
`ErrorOr` is already `nodiscard` - I'm not against stating it explicitly here again, but then I'm not sure what the point of marking classes themselves as `nodiscard` is :)

Reply:
That is a better way of doing it. I will remove it from these spots.